react-props/my-app/src/Image.tsx

```diff
@@ -0,0 +1,15 @@
+import './Image.css';
+
+type Image = {
```

Comment:
It's best if your type and function names are different. I find the code more readable if you just use `type Props` for all your components. That name makes it very clear that this type is describing the component's properties.